getting-started/kubernetes/customizations/imageset.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Configure Image deployment by digest
+description: Specify the digests the operator will use to deploy images
+---
+
+{% assign operator = site.data.versions.first.tigera-operator %}
+
+### Big picture
+
+Enable deploying images by docker registry digest.
+
+### Value
+
+Some deployments have strict security requirements that require deploying images by immutable digest instead of tags.
+{{site.prodname}} will not update existing tags but using an immutable digest allows specific images to be reviewed
+and verified by security teams.
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+* **ImageSet**
+
+### Concepts
+
+#### Docker repository
+
+A docker respository provides access to docker images referenced by tags or digest.
+
+#### Image tag
+
+Versioned docker images are typically referenced by a tag which is appended to an image reference. Example: `<repo>/<image>:<tag>`. Docker image tags are typically not expected be changed or updated, but this is not required or enforced.
+
+#### Image digest
+
+Docker images, when added to a docker repository, have a unique hash created that can be used to pull a specific version of an image that cannot be changed or updated.
+
+### FAQ
```

Comment:
Maybe FAQ should go after the how-to section?

Reply:
I was following our How to template. I agree though, it feels odd to me to include the FAQ here.

Reply:
Propose this:

Move FAQ to How to list:

   Verify imageset
   Update operator deployment with digest
   Create an image manifest
   Create new image set when upgrading or downgrading (from FAQ)
   Troubleshoot (other two FAQ items)

Reply:
After chatting with @bmckercher123 I'm goingto split the HowTo section with a numbered list and then Other tasks. The Upgrade/downgrade and the other 2 FAQ items will be moved to Other tasks.